### DISCUSSION TITLE
How can I call Android's SDK and jar packages?

### DISCUSSION THREAD

**Discussion by bbhxwl** [AvaloniaUI/Avalonia#15917] [General]
How can I call Android's SDK and jar packages?

**Comment by maxkatz6** [comment]
You can call Android SDKs freely from your "net8.0-android" project.
Embedding external android libraries is harder, but should be possible: https://www.yoctopuce.com/EN/article/how-to-use-our-android-library-with-xamarin

Note, it's not handled by Avalonia. .NET SDK provides these integrations.

**Reply by maxkatz6** [reply]
https://learn.microsoft.com/en-us/previous-versions/xamarin/android/deploy-test/building-apps/build-items#androidlibrary
